src/resolve/resolved-manager.c

```diff
@@ -1054,6 +1054,67 @@ static int manager_ipv6_send(
+        int ifindex, r;
+        _cleanup_(json_variant_unrefp) JsonVariant *array = NULL;
+        Varlink *connection = NULL;
+
+        assert(m);
+
+        DNS_ANSWER_FOREACH_IFINDEX(rr, ifindex, answer) {
+                _cleanup_(json_variant_unrefp) JsonVariant *entry = NULL;
+                int family;
+
+                if (rr->key->type == DNS_TYPE_A) {
+                        struct in_addr *addr = &rr->a.in_addr;
+                        family = AF_INET;
+                        r = json_build(&entry,
+                                JSON_BUILD_OBJECT(JSON_BUILD_PAIR_CONDITION(ifindex > 0, "ifindex", JSON_BUILD_INTEGER(ifindex)),
+                                                        JSON_BUILD_PAIR("family", JSON_BUILD_INTEGER(family)),
+                                                        JSON_BUILD_PAIR("address", JSON_BUILD_BYTE_ARRAY(addr, FAMILY_ADDRESS_SIZE(family))),
+                                                        JSON_BUILD_PAIR("type", JSON_BUILD_STRING("A"))));
+                } else if (rr->key->type == DNS_TYPE_AAAA) {
+                        struct in6_addr *addr6 = &rr->aaaa.in6_addr;
+                        family = AF_INET6;
+                        r = json_build(&entry,
+                                JSON_BUILD_OBJECT(JSON_BUILD_PAIR_CONDITION(ifindex > 0, "ifindex", JSON_BUILD_INTEGER(ifindex)),
+                                                        JSON_BUILD_PAIR("family", JSON_BUILD_INTEGER(family)),
+                                                        JSON_BUILD_PAIR("address", JSON_BUILD_BYTE_ARRAY(addr6, FAMILY_ADDRESS_SIZE(family))),
+                                                        JSON_BUILD_PAIR("type", JSON_BUILD_STRING("AAAA"))));
+                } else
+                        continue;
+
+                if (!canonical)
+                        canonical = dns_resource_record_ref(rr);
+
+                r = json_variant_append_array(&array, entry);
+                if (r < 0)
+                        return log_error_errno(r, "Failed to append notification entry to array: %m");
+        }
+
+        if (json_variant_is_blank_object(array))
+                return 0;
+
+        if (!canonical)
+                return log_error_errno(SYNTHETIC_ERRNO(EBADMSG), "Failed to determine canonical name: %m");
+
+        r = dns_name_normalize(dns_resource_key_name(canonical->key), 0, &normalized);
+        if (r < 0)
+                return log_error_errno(r, "Failed to get hostname: %m");
+
+        SET_FOREACH(connection, m->varlink_subscription) {
+                r = varlink_notifyb(connection, JSON_BUILD_OBJECT(JSON_BUILD_PAIR("addresses",
+                                                                  JSON_BUILD_VARIANT(array)),
+                                                                  JSON_BUILD_PAIR("name", JSON_BUILD_STRING(query_name))));
+                if (r < 0)
+                        log_error_errno(r, "Failed to send notification: %m");
```

Comment:
"Failed to send notification, ignoring:"

Reply:
not addressed

Reply:
Fixed in latest patch